packages/dashboards/addon/components/dashboard-action-list.hbs

```diff
@@ -1,82 +1,74 @@
 {{!-- Copyright 2020, Yahoo Holdings Inc. Licensed under the terms of the MIT license. See accompanying LICENSE.md file for terms. --}}
-<ul class="dashboard-action-list actions" ...attributes>
+<div class="dashboard-action-list flex-1" ...attributes>
   {{#with @item as | dashboard |}}
     {{!-- Clone action enabled at all times --}}
-    <li class="action">
-      <LinkTo class="clone" @route="dashboards.dashboard.clone" @model={{dashboard.id}}>
-        <NaviIcon @icon="copy" class="navi-icon__copy" />
-        <EmberTooltip @text="Clone the dashboard"/>
-      </LinkTo>
-    </li>
+    <LinkTo class="clone" @route="dashboards.dashboard.clone" @model={{dashboard.id}}>
+      <DenaliIcon @size="small" @icon="copy-file" class="navi-icon__copy p-r-5 link" />
+      <EmberTooltip @text="Clone the dashboard"/>
+    </LinkTo>
 
     {{#if (feature-flag "enableDashboardExport")}}
       {{!-- Export action enabled if the dashboard is valid --}}
-      <li class="action">
-        <EmberTooltip @text="Export the dashboard"/>
-        <DashboardActions::Export
-          class="export btn {{unless dashboard.validations.isTruelyValid "disabled"}}"
-          @dashboard={{dashboard}}
-          @disabled={{not dashboard.validations.isTruelyValid}}
-        >
-          <NaviIcon @icon="download" class="navi-icon__download" />
-        </DashboardActions::Export>
-      </li>
+      <EmberTooltip @text="Export the dashboard"/>
+      <DashboardActions::Export
+        class="export {{unless dashboard.validations.isTruelyValid "disabled"}}"
+        @dashboard={{dashboard}}
+        @disabled={{not dashboard.validations.isTruelyValid}}
+      >
+        <DenaliIcon @size="small" @icon="download" class="navi-icon__download p-r-5 link" />
```

Comment:
Looks like we are missing`is-disabled` class in the icons when the buttons are disabled.